lib/osc-loader.js

```diff
@@ -0,0 +1,93 @@
+'use babel'
+
+import * as path from 'path';
+import { spawn } from 'child_process';
+import { EventEmitter } from 'events';
+
+const oscPortProperty = 'tidalcycles.oscEvalPort';
+const oscAddressProperty = 'tidalcycles.oscEvalAddress';
+
+export default class OscLoader extends EventEmitter {
+
+    constructor(tidalRepl) {
+        super();
+        console.log("listening on port", atom.config.get(oscPortProperty));
+        this.port = atom.config.get(oscPortProperty);
+        this.server = spawn(
+            'node',
+            [path.resolve(__dirname, 'osc-server.js'), this.port.toString()],
```

Comment:
Maybe will be better to implement osc server as a class so it will be possibile to start it and stop it like
```
oscServer.start(port)
```
without using the child_process class to invoke external file.

Reply:
I believe this is a great idea. 
Done